assets/js/editor.js

```diff
@@ -108,7 +125,8 @@ $gp.editor = (
 					.on( 'click', 'button.reject', $gp.editor.hooks.set_status_rejected )
 					.on( 'click', 'button.fuzzy', $gp.editor.hooks.set_status_fuzzy )
 					.on( 'click', 'button.ok', $gp.editor.hooks.ok )
-					.on( 'keydown', 'tr.editor textarea', $gp.editor.hooks.keydown );
+					.on( 'keydown', 'tr.editor textarea', $gp.editor.hooks.keydown )
+					.on( 'change keyup paste', 'tr.editor textarea.foreign-text', $gp.editor.hooks.update_count );
```

Comment:
```suggestion
					.on( 'input', 'tr.editor textarea.foreign-text', $gp.editor.hooks.update_count );
```

`input` should catch all these events.